tests/dtypes/test_array.py

```diff
@@ -0,0 +1,67 @@
+        example                     |                                            |
+                                    |                                            |
+          |   |   | Language        |                                            |
+          |   |   |   | name        | string                                     |
+          |   |   |   | countries[] | backref                                    | Country
+                                    |                                            |
+          |   |   | Country         |                                            |
+          |   |   |   | name        | string                                     |
+          |   |   |   | languages   | array                                      | CountryLanguage[country, language]
+          |   |   |   | languages[] | ref                                        | Language
+                                    |                                            |
+          |   |   | CountryLanguage |                                            |
+          |   |   |   | language    | ref                                        | Language
+          |   |   |   | country     | ref                                        | Country
+    '''
+    create_tabular_manifest(tmp_path / 'manifest.csv', table)
+    manifest = load_manifest(rc, tmp_path / 'manifest.csv')
+    assert manifest == table
+
+
+def test_prop_array_with_custom_without_properties(rc, tmp_path):
+    table = '''
+        d | r | b | m | property    | type                                       | ref
+        example                     |                                            |
+                                    |                                            |
+          |   |   | Language        |                                            |
+          |   |   |   | name        | string                                     |
+          |   |   |   | countries[] | backref                                    | Country
+                                    |                                            |
+          |   |   | Country         |                                            |
+          |   |   |   | name        | string                                     |
+          |   |   |   | languages   | array                                      | CountryLanguage
+          |   |   |   | languages[] | ref                                        | Language
+                                    |                                            |
+          |   |   | CountryLanguage |                                            |
+          |   |   |   | language    | ref                                        | Language
+          |   |   |   | country     | ref                                        | Country
+    '''
+    create_tabular_manifest(tmp_path / 'manifest.csv', table)
+    manifest = load_manifest(rc, tmp_path / 'manifest.csv')
+    assert manifest == table
```

Comment:
Add tests for writes and reads with backref type, when reading and writing backref tipe should not be used. Fo example, when you read data, all properties of backref type should not be visible in the result. And on write action, data for backref property can't be given.